Library/Homebrew/global.rb

```diff
@@ -82,7 +84,12 @@ def failed?
     end
 
     def args
-      @args ||= CLI::Args.new
+      @args ||= CLI::Args.new(global: true)
+      unless @args.global || @printed_args_warning
```

Comment:
Can this be reworked to be an `if` or an early return and multiple conditions? `unless ||` is hard to follow. Thanks!

Reply:
I'm not sure when `global` will ever be `true` since `Homebrew.args` is assigned in `brew.rb` with `global: false`.